consensus/istanbul/backend/announce.md

```diff
@@ -0,0 +1,140 @@
+# Celo Announce Protocol
+
+While here referred as a protocol by itself, Announce is technically a subset of the Istanbul protocol, on the [RLPx] transport, that facilitates Validators to privately discover the eNodeURL of other Validators in the network, using the underlying p2p eth network. This is used by Istanbul to have direct connections between Validators, speeding up the consensus phase of block generation.
+
+It is important to note that a validator's `eNodeURL` is never shared among third parties, but directly published to specific peers. Therefore it is allowed for a validator to publish different external facing `eNodeURL` values to different validator peers.
```

Comment:
```suggestion
It is important to note that a validator's `eNodeURL` is never shared among third parties, but directly published to specific peers. Therefore it is allowed for a validator to publish different external facing `eNodeURL` values to different validator peers.

An `eNodeURL` has the following format:

`enode://<hex node id>@<IP/hostname>:<tcp port>?discport=<udp discovery port>`

Where `<hex node id>` is the node's serialized and hex encoded ecdsa public key. The URL parameter `discport` should only be specified if the udp discovery port differs from the tcp port.

Some example `eNodeURLs` (with partially elided hex encoded public keys):
\```
enode://517318.......607ff3@127.0.0.1:34055
enode://e5e2fdf.......348ce8@127.0.0.1:33503?discport=22042
\```
```

Note: I had to escape the code block inside the suggestion, so you will need to tweak this a bit.